script.js

```diff
@@ -1,4 +1,312 @@
+  displayPlayerHand = [];
+  displayComputerHand = [];
+
+  var output = document.getElementById("output-div");
+  output.innerText = `The game has been reset. Press Start to play a new game.`;
+});
+
+// Create a deck of cards
+var makeDeck = function () {
+  // Initialise an empty deck array
+  var cardDeck = [];
+  // Initialise an array of the 4 suits in our deck. We will loop over this array.
+  var suits = ["♥️", "♦️", "♣️", "♠️"];
+
+  // Loop over the suits array
+  var suitIndex = 0;
+  while (suitIndex < suits.length) {
+    // Store the current suit in a variable
+    var currentSuit = suits[suitIndex];
+
+    // Loop from 1 to 13 to create all cards for a given suit
+    // Notice rankCounter starts at 1 and not 0, and ends at 13 and not 12.
+    // This is an example of a loop without an array.
+    var rankCounter = 1;
+    while (rankCounter <= 13) {
+      // By default, the card name is the same as rankCounter
+      var cardName = rankCounter;
+      var val = rankCounter;
+
+      // If rank is 1, 11, 12, or 13, set cardName to the ace or face card's name
+      if (cardName == 1) {
+        cardName = "ace";
+      } else if (cardName == 11) {
+        cardName = "jack";
+        val = 10;
+      } else if (cardName == 12) {
+        cardName = "queen";
+        val = 10;
+      } else if (cardName == 13) {
+        cardName = "king";
+        val = 10;
+      }
+
+      // Create a new card with the current name, suit, and rank
+      var card = {
+        name: cardName + " of " + currentSuit,
+        suit: currentSuit,
+        rank: rankCounter,
+        value: val,
+      };
+
+      // Add the new card to the deck
+      cardDeck.push(card);
+
+      // Increment rankCounter to iterate over the next rank
+      rankCounter += 1;
+    }
+
+    // Increment the suit index to iterate over the next suit
+    suitIndex += 1;
+  }
+
+  // Return the completed card deck
+
+  return cardDeck;
+};
+
+// Shuffle card code
+// Get a random index ranging from 0 (inclusive) to max (exclusive).
+var getRandomIndex = function (max) {
+  return Math.floor(Math.random() * max);
+};
+
+// Shuffle the elements in the cardDeck array
+var shuffleCards = function (cardDeck) {
+  // Loop over the card deck array once
+  var currentIndex = 0;
+  while (currentIndex < cardDeck.length) {
+    // Select a random index in the deck
+    var randomIndex = getRandomIndex(cardDeck.length);
+    // Select the card that corresponds to randomIndex
+    var randomCard = cardDeck[randomIndex];
+    // Select the card that corresponds to currentIndex
+    var currentCard = cardDeck[currentIndex];
+    // Swap positions of randomCard and currentCard in the deck
+    cardDeck[currentIndex] = randomCard;
+    cardDeck[randomIndex] = currentCard;
+    // Increment currentIndex
+    currentIndex = currentIndex + 1;
+  }
+
+  // Return the shuffled deck
+  return cardDeck;
+};
+
+// Helper functions for scores
+var playerBlackJack = function (playerScore) {
```

Comment:
For you helper functions for scores, do you realise that you're repeating the same logic for the player and the computer?
You're passing in arguments correctly but I think because of the name you gave it, you might be confusing yourself. Try renaming the parameters to score. 
Also, functions that return booleans should have names that resolve to a true or false answer

e.g. 
```javascript
function isBlackJack(score) {
    return score == 21
}
```